[Android] Compile Error

I'm getting a Compile Error with latest react-native-braintree-xplat (2.1.0) and React Native (0.34.1).

I created a Push Request to solve this issue but it will most probably break older code bases. What do you suggest?

Error Output:
```
:react-native-braintree-xplat:compileReleaseJavaWithJavac
/Users/jyotipuri/humble/gostudent/mobile-app/node_modules/react-native-braintree-xplat/android/src/main/java/com/pw/droplet/braintree/Braintree.java:22: error: Braintree is not abstract and does not override abstract method onActivityResult(Activity,int,int,Intent) in ActivityEventListener
public class Braintree extends ReactContextBaseJavaModule implements ActivityEventListener {
       ^
/Users/jyotipuri/humble/gostudent/mobile-app/node_modules/react-native-braintree-xplat/android/src/main/java/com/pw/droplet/braintree/Braintree.java:108: error: method does not override or implement a method from a supertype
  @Override
  ^
2 errors
:react-native-braintree-xplat:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-braintree-xplat:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 33.236 secs
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/android-setup.html
```

--- a/android/src/main/java/com/pw/droplet/braintree/Braintree.java
+++ b/android/src/main/java/com/pw/droplet/braintree/Braintree.java
@@ -106,7 +106,7 @@ public class Braintree extends ReactContextBaseJavaModule implements ActivityEve
   }
 
   @Override
-  public void onActivityResult(final int requestCode, final int resultCode, final Intent data) {
+  public void onActivityResult(Activity activity, final int requestCode, final int resultCode, final Intent data) {
     if (requestCode == PAYMENT_REQUEST) {
       switch (resultCode) {
         case Activity.RESULT_OK:

